acceptance/analyzer_test.go

```diff
@@ -960,25 +987,24 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 						// Don't attempt to remove the image, as it's stored in the test registry, which is ephemeral.
 						// Attempting to remove the image sometimes produces `No such image` flakes.
 
-						it("restores cache metadata", func() {
+						it("throw read/write error", func() {
 							h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7 does not read from the cache")
-							output := h.DockerRunAndCopy(t,
-								containerName,
-								copyDir,
-								ctrPath("/layers"),
+							cmd := exec.Command(
+								"docker", "run", "--rm",
+								"--network", registryNetwork,
+								"--env", "CNB_PLATFORM_API="+platformAPI,
+								"--name", containerName,
 								analyzeImage,
-								h.WithFlags(
-									"--network", registryNetwork,
-									"--env", "CNB_PLATFORM_API="+platformAPI,
-								),
-								h.WithArgs(
-									ctrPath(analyzerPath),
-									"-cache-image", noAuthRegCacheImage,
-									"some-image",
-								),
-							)
+								ctrPath(analyzerPath),
+								"-cache-image",
+								noAuthRegCacheImage,
```

Comment:
I'm still a bit confused as to why we don't have read / write access to the "no auth" registry... should this be called `readOnlyRegCacheImage`?

Reply:
```suggestion
								readOnlyRegCacheImage,
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/2228fae52b7d0bbeeefe01e882486b1d26b30f1c)